jobs/service-fabrik-broker/spec

```diff
@@ -151,6 +156,16 @@ properties:
   broker_drain_message:
     description: "Drain message that is updated into broker maintenance state. If undefined, then during drain maintenance state is not updated"
     default: "BROKER_DRAIN_INITIATED"
+  
```

Comment:
just a question , for a component who does not want to use rate limiting feature will these things be a must have in the manifest ?